canonical-web-and-design/vanilla-framework
# Add .row-breakout to enable layouts that spill into the page margins

Add a .row-breakout class that allows content to bleed into the page margins (up to a certain point).

QA:
 -Go to /docs/patterns/grid#breaking-out-of-the-grid
- Proof read copy, suggest improvements if something doesn't make sense
- Try importing the plugin with modified arguments, as described in the copy, ensure it works
- QA the following examples on different screen sizes: 
1) left aside: 
docs/examples/patterns/grid/grid-breakout-left-aside

2) right aside:
/docs/examples/patterns/grid/grid-breakout-right-aside

- Try to break it in different ways and report back :)
